package.json

```diff
@@ -66,7 +66,8 @@
         "jest": "^29.7.0",
         "jsdoc": "^4.0.2",
         "jsdoc-to-markdown": "^8.0.0",
-        "npm-run-all": "^4.1.5"
+        "npm-run-all": "^4.1.5",
+        "uglify-js-export": "^3.19.2"
```

Comment:
Should this be in dependencies, not devDependencies? It is used in the running code, no?

Reply:
You are right. It should be in dependencies. so I update it.